test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -83,6 +89,12 @@ void
 call_log_manager::call_log_close_session(const json &call_log_entry)
 {
     const std::string session_id = call_log_entry["session_id"].get<std::string>();
+
+    /* If there is a failure in opening a session, there is no work to do. */
```

Comment:
`opening` should be `closing`

Reply:
Updated in [d6a71e6](https://github.com/wiredtiger/wiredtiger/pull/8167/commits/d6a71e6cbe2d01322377362465023199d723914c)